release/dist/dist.go

```diff
@@ -43,6 +43,8 @@ type Build struct {
 	Tmp string
 	// Go is the path to the Go binary to use for building.
 	Go string
+	// Yarn is the path to the yarn binary to use for building.
+	Yarn string
```

Comment:
Maybe a small doc that this is only used to build the web client.

Reply:
done